scipy/stats/morestats.py

```diff
@@ -1795,6 +1798,9 @@ def rootfunc(ab, xj, N):
         logsf = distributions.gumbel_r.logsf(w)
         sig = array([25, 10, 5, 2.5, 1])
         critical = around(_Avals_gumbel / (1.0 + 0.2/sqrt(N)), 3)
+    elif dist == 'weibull_min':
+        x = np.log(np.divide(1, x))
+        return anderson(x, dist="gumbel_r")
```

Comment:
I guess the use of the transformation is a valid approach. however, the critical values are not adjusted for the Weibul distribution and this might be confusing for the user. Can you check the critical values returned in goftest in R?

If we use this approach, the documentation needs to be very clear about this.